docs/detections/rules-ui-create.asciidoc

```diff
@@ -355,6 +355,11 @@ also affect this rule.
 .. *Building block* (optional): Select to create a building-block rule. By
 default, alerts generated from a building-block rule are not displayed in the
 UI. See <<building-block-rule>> for more information.
+.. *Indicator prefix override*: Specify the default threat indicator path (`threat_indicator_path`) that the indicator match rule will use when obtaining threat indicator data from the default threat intelligence index. This data is used to enrich indicator match alerts with metadata about matched threat indicators. The default value for this field is `threat.indicator`. If your threat indicator data is at a different location, update this setting to ensure the correct field prefix is used so alert enrichment can still be performed.
+
++
+NOTE: When upgrading to {stack} version 8.x, the default threat indicator path of an indicator match rule created in {stack} 7.x is automatically updated to `threatintel.indicator`. This allows the rule to continue using indicator data ingested by {filebeat} version 7.x. To make a rule use threat indicator data ingested from {agent} or {filebeat} version 8.x, enter the `threat.indicator` value into the *Indicator prefix override* field. Separate rules must be created to query threat intelligence indices created by {filebeat} version 7.x and version 8.x because different default threat indicator path values are required.
```

Comment:
first sentence: the value is automatically updated to `threatintel.indicator` IF it wasn't previously specified. if the users had provided a custom value that won't be overwritten.

Reply:
third sentence: users won't have to manually enter `threat.indicator` if they are creating a new rule, we auto-populate that by default in 8.x. If they had created a rule in 7.x with filebeat, and they upgrade both kibana and filebeat to 8.0, only then they would have to come back to their old rule and update `threatintel.indicator` to `threat.indicator`